tests/bootstrap.php

```diff
@@ -82,11 +85,15 @@ public function install_wc() {
 
 		WC_Install::install();
 
-		// Reload capabilities after install, see https://core.trac.wordpress.org/ticket/28374
+		// Initialize the WC API extensions.
```

Comment:
This comment updated (or removed as to me it seems straigthforward to understand what happening in the two lines below just by reading the class and method names).